.github/workflows/semantic.yaml

```diff
@@ -0,0 +1,35 @@
+name: "PR Lint"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  main:
+    name: Validate PR title
```

Comment:
We can use https://github.com/Ezard/semantic-prs and ask ASF INFRA to enable the App for this repo.

It helps us save GitHub Actions workload.